scipy/stats/_levy_stable/__init__.py

```diff
@@ -575,32 +581,33 @@ def _fitstart_S1(data):
         [0, -0.061, -0.279, -0.659, -1.198]]
     # fmt: on
 
-    psi_1 = interpolate.interp2d(
-        nu_beta_range, nu_alpha_range, alpha_table, kind="linear"
-    )
+    j = np.argsort(alpha_range)
+    alpha_range_r = np.asarray(alpha_range)[j]
+    nu_c_table_r = np.asarray(nu_c_table)[j, :]
+    nu_zeta_table_r = np.asarray(nu_zeta_table)[j, :]
+
+    psi_1 = RectBivariateSpline(nu_beta_range, nu_alpha_range_r,
+                                alpha_table_r.T, kx=1, ky=1, s=0)
```

Comment:
```suggestion
                                alpha_table_r.T, kx=1, ky=1)
```
This is the default. Just a thought, unless you wanted to emphasize it.

Reply:
Yeah, it's inconsistent across various FITPACK wrappers (e.g. SmoothBivariateSpline has a differet default), so better be explicit and save a trip to the docs and then to the source to check that the docs are not stale.